src/site/content/en/blog/conversion-measurement/index.md

```diff
@@ -0,0 +1,696 @@
+
+Now, the browser knows that it needs to inform the adtech server of this conversion—more
+specifically, the browser must inform the `reportingorigin` that is specified in both the `<a>`
+element and in the pixel request (`adtech.example`).
+
+To do so, the browser schedules to send a **conversion report**, a blob of data containing the click
+data (from the publisher's site) and the conversion data (from the advertiser's). For this example,
+the user converted one day after click. So the report is scheduled to be sent on the next day, at
+the two-day-after-click mark if the browser is running.
+
+### Sending the report (steps 10 and 11)
+
+<figure class="w-figure">
+  <img src="./steps-3.jpg" alt="Diagram: browser sending the report">
+</figure>
+
+Once the scheduled time to send the report is reached, the browser sends the **conversion report**:
+it sends an HTTP POST to the reporting origin that was specified in the `<a>` element
+(`adtech.example`). For example:
+
+`https://adtech.example/.well-known/register-conversion?impression-data=776f09351f5809c5&conversion-data=2&credit=100`
+
+Included as parameters are:
+
+- The data associated with the original ad click (`impression-data`).
+- The data associated with a conversion, [potentially noised](link to above).
+- The conversion credit attributed to the click. This API follows a **last-click attribution**
+  model: the most recent matching ad click is given a credit of 100, all other matching ad clicks
+  are given a credit of 0.
+
+As the adtech server receives this request, it can pull the `impression-data` and `conversion-data`
+from it, i.e. the conversion report:
+
+```json
+{"impression-data": "776f09351f5809c5", "conversion-data": 3, "credit": 100}
+```
+
+### Subsequent conversions and expiry
+
+Later on, the user may convert again—for example by purchasing a tennis racket on
+`advertiser.example` to go alongside their shoes. A similar flow takes place:
+
+- The adtech server sends a conversion request to the browser.
+- The browser matches this conversion with the ad click, schedules a report, and sends it to the
+  adtech server later on.
+
+After `impressionexpiry`, conversions for this ad click stop being counted and the ad click is
+deleted from browser storage.
+
+## Use cases
+
+### What is currently supported
+
+- Measure click-through conversions: determine which ad clicks lead to conversions, and access
+  coarse information about the conversion.
+- Gather data to optimize ad selection, for example by training machine learning models.
+
+### What is not supported yet
+
+The following features aren't supported yet, but may be in future iterations of this API, or in the
+[Aggregate Conversion Measurement
+API](https://github.com/WICG/conversion-measurement-api/blob/master/AGGREGATE.md):
+
+- View-through conversion measurement.
+- [Multiple reporting endpoints](https://github.com/WICG/conversion-measurement-api/issues/29).
+- [Web conversions that started in a native
```

Comment:
```suggestion
- [Web conversions that started in an iOS/Android
```
FYI @maudnals `native` is a "do not use" word now (based on P&E's efforts to stop using words with problematic histories)